core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -139,20 +139,28 @@ private[spark] class Executor(
   private val executorPlugins: Seq[ExecutorPlugin] = {
     val pluginNames = conf.get(EXECUTOR_PLUGINS)
     if (pluginNames.nonEmpty) {
-      logDebug(s"Initializing the following plugins: ${pluginNames.mkString(", ")}")
+      logDebug("Initializing the following executor plugin(s): " +
```

Comment:
No need to change this; this is the "Executor" class, so it's kinda assumed they're executor plugins.